private.php

```diff
@@ -704,8 +705,8 @@
 		{
 			$optionschecked['readreceipt'] = 'checked="checked"';
 		}
-		$to = htmlspecialchars_uni($mybb->get_input('to'));
-		$bcc = htmlspecialchars_uni($mybb->get_input('bcc'));
+		$to = htmlspecialchars_uni(implode(', ',array_unique(array_map('trim', explode(',', $mybb->get_input('to'))))));
```

Comment:
I know I might seem pedant here, but there should be a space between `implode(', ',` and `array_unique` :)

Reply:
Done.